tailwind.config.cjs

```diff
@@ -2,6 +2,7 @@ const colors = require('tailwindcss/colors');
 const process = require('process');
 
 module.exports = {
+  important: true,
```

Comment:
looks like the spinners are no longer spinning when i send a transaction. might it be because of this?